### PR TITLE
fixes #14850 - drop foreman 1.11 support

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: lib/foreman_remote_execution/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ForemanRemoteExecution
-  VERSION = '0.3.0'
+  VERSION = '1.0.0'
```
Comment (stbenjam) [inline lib/foreman_remote_execution/version.rb]: Did we settle on this being 1.0 or 0.4?

I wasn't 100% sure if we were going to call this 1.0 and then start following http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies going forward.

Comment (iNecas) [inline lib/foreman_remote_execution/version.rb]: I'm ok with 1.0

Comment (ares) [inline lib/foreman_remote_execution/version.rb]: I'm ok with both as long as we have redmine versions :-)

Comment (adamruzicka) [inline lib/foreman_remote_execution/version.rb]: ^ This

**Comment by stbenjam** [comment]
This should be good to go now that we've branched for 0.3-stable.

**Comment by ares** [comment]
The support was dropped by #189, merging, thanks @stbenjam

**Pull request merged**